.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: python
+python:
+- '2.7'
+install:
+- pip install -r requirements.txt
+- pip install coveralls
```

Comment:
If you have coveralls included in your requirements.txt no need to run another command to install it